sql/privilege.md

```diff
@@ -6,80 +6,13 @@ category: user guide
 
 # Privilege Management
 
-TiDB's privilege management system is implemented according to the privilege management system in MySQL. It supports most of the syntaxes and privilege types in MySQL. If you find any inconsistency with MySQL, feel free to [open an issue](https://github.com/pingcap/tidb/issues/new/choose).
+TiDB's privilege management system is implemented according to the privilege management system in MySQL. It supports most of the syntaxes and privilege types in MySQL.
```

Comment:
I think it's important to nuance what the support looks like. TiDB is mostly compatible with MySQL 5.7, but supports roles as well (MySQL 8.0).  It does not support [dynamic privileges](https://dev.mysql.com/doc/refman/8.0/en/privileges-provided.html#privileges-provided-dynamic) (8.0) to my knowledge.

```suggestion
TiDB supports MySQL 5.7's privilege management system, including the syntax and privilege types. Starting with TiDB 3.0, support for SQL Roles is also available.
```

PTAL @tiancaiamao

Reply:
Yes, you're right